### PR TITLE
Add knplabs/packagist-api

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+services:
+    Packagist\Api\Client: null
```
Comment (Nyholm) [inline knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml:2]: Could you elaborate on this? Dont you need it to be auto wired?

Comment (OskarStark) [reply knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml:2]: I want to be able to autowire this serve, currently I do it like this:
https://github.com/sonata-project/dev-kit/blob/fa081ebaeaec0de4b140d87ec2b926c1d706c958/config/services.yaml#L39-L42

Comment (Nyholm) [reply knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml:2]: You need to add: 

```yaml
services:
    Packagist\Api\Client:
        autowire: true
```

Just as you did in the `_default` section of the file you linked.

Comment (OskarStark) [reply knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml:2]: Are you sure, when using this:
```
# config/test.yaml
services:
    Packagist\Api\Client: ~
```

it works 🤔 

CleanShot 2020-12-03 at 15 21 54

Comment (Nyholm) [reply knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml:2]: Oh.. I thought that class had some required constructor parameters.

Okey, then it works with the current implementation

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**